bindings/influx/influx.go

```diff
@@ -16,21 +16,39 @@ package influx
 import (
 	"context"
 	"encoding/json"
-	"errors"
 	"fmt"
 
 	influxdb2 "github.com/influxdata/influxdb-client-go"
 	"github.com/influxdata/influxdb-client-go/api"
+	"github.com/pkg/errors"
 
 	"github.com/dapr/components-contrib/bindings"
 	"github.com/dapr/kit/logger"
 )
 
+const queryOperation bindings.OperationKind = "query"
```

Comment:
please combine const field.

Reply:
I think the groups of what are not similar in type although they are the same declared variable, so it is intentional.

Same as L44. This is used to check whether a structure implements an interface
```go
var _ bindings.OutputBinding = &Influx{}
```
ref:[uber-go Guide](https://github.com/uber-go/guide/blob/master/style.md#group-similar-declarations)